client/src/components/UpcomingTripsCard/UpcomingTripsCard.js

```diff
@@ -0,0 +1,37 @@
+
+import React from 'react';
+import "./UpcomingTripsCard.css";
+import "../../App.css";
```

Comment:
You don't need to import `App.css` as create react app will merge all the CSS files together